<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Login App</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_SignIn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Sign In</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>199</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_Username">
        <item>
         <widget class="QLabel" name="label_username">
          <property name="text">
           <string>Username</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_username</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_username"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_Password">
        <item>
         <widget class="QLabel" name="label_password">
          <property name="text">
           <string>Password</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_password</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_password">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_login">
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="profileImage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_serverload">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Server Load: </string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_serverload">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
